sirepo/pkcli/nersc_test.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+"""NERSC test suite
+
+:copyright: Copyright (c) 2023 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+from pykern.pkcollections import PKDict
+from pykern.pkdebug import pkdc, pkdexc, pkdp, pkdlog
+import pykern.pkio
+import pykern.pkjson
+import pykern.pksubprocess
+import pykern.pkjinja
+import sirepo.sim_data
+import sirepo.const
+import sirepo.resource
+import shutil
+
+
+_SEQUENTIAL_TEST_BASH_FILE = "sequential_test.sh"
+_SEQUENTIAL_TEST_BASH_TEMPLATE = _SEQUENTIAL_TEST_BASH_FILE + ".jinja"
+_NERSC_TEST_DIR = "nersc_test/"
+_RUN_DIR = "sirepo_run_dir"
+_SEQUENTIAL_TEST_JSON = "nersc_sequential.json"
+_SEQUENTIAL_RESULT_FILE = "nersc_sequential.log"
+
+
+def sequential():
+    """Sequential testing of job_cmd for nersc. Generates bash script
+    that invokes job_cmd on a srw in.json with one MPI core.
+    """
+    try:
+        s = pykern.pkio.py_path(_RUN_DIR)
+        pykern.pkio.unchecked_remove(s)
+        s.ensure(dir=True)
+        o = s.join(_SEQUENTIAL_RESULT_FILE)
+        shutil.copyfile(
```

Comment:
The copy should not be necessary. Rather, use pkjinja.render_resource. There's a subtlety here, which creates a problem that we need for another issue. We should use sirepo.resource, but it doesn't have a render_file. It should so create an issue for that. sirepo.resource manages package_path, which we need to conform to.